app/src/main/java/com/matryoshka/projectx/Constants.kt

```diff
@@ -9,8 +9,27 @@ object SavedStateKey {
 
 object NavArgument {
     const val ARG_EMAIL = "email"
+    const val ARG_USER_NAME = "user_name"
     const val ARG_LOCATION = "location"
     const val ARG_INTEREST_ID = "interest_id"
     const val ARG_EVENT_ID = "event_id"
     const val ARG_EVENT = "event"
 }
+
+object MattyApiPath {
+    const val SEND_REGISTER_CODE_PATH = "registration/code"
```

Comment:
```suggestion
    const val REGISTER_CODE_PATH = "registration/code"
```
Я бы не стал добавлять действие в название эндпоинта
По этой точке мог бы также находится метод для рефреша кода с query-параметром типа registration/code?refresh

Reply:
Исправил